### ISSUE TITLE
When I start the one click installer win_start.bat displays some text and then closes out. Crashes Windows Terminal 

### ISSUE THREAD

**Issue by internetuser1234** [Sanster/lama-cleaner#113]
The pip install works, but the  installer win_start.bat crashes Windows Terminal.

**Comment by Sanster** [comment]
Did you click `win_config.bat` first? 

If you have executed it, please execute `win_start.bat` at the command line to see what error is reported

**Comment by ffdown** [comment]
1. win_config.exe
2. win_start.exe
3. Crash "Traceback (most recent call last):
  File "E:\AI\lama-cleaner\installer\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "E:\AI\lama-cleaner\installer\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "E:\AI\lama-cleaner\installer\Scripts\lama-cleaner.exe\__main__.py", line 7, in <module>
  File "E:\AI\lama-cleaner\installer\lib\site-packages\lama_cleaner\__init__.py", line 9, in entry_point
    main(args)
  File "E:\AI\lama-cleaner\installer\lib\site-packages\lama_cleaner\server.py", line 233, in main
    ui = FlaskUI(
TypeError: FlaskUI.__init__() got an unexpected keyword argument 'host'"

**Comment by CCpt5** [comment]
Similar issue here on a new workstation I'm trying to install it on:


------------------Environment information-------------------
git version 2.34.1.windows.1
conda 22.9.0
/d/Downloads/lama-cleaner-win/lama-cleaner/installer/python
Python 3.10.6
/d/Downloads/lama-cleaner-win/lama-cleaner/installer/Scripts/pip
pip 22.2.2 from D:\Downloads\lama-cleaner-win\lama-cleaner\installer\lib\site-packages\pip (python 3.10)
diffusers              0.6.0
lama-cleaner           0.24.5
opencv-python          4.6.0.66
torch                  1.12.1+cu113
------------------------------------------------------------
[14:37:48] INFO     Load config from config.json                                                            tasks.py:111           INFO     Using random port: 1151                                                                 tasks.py:120Fetching 15 files: 100%|##########| 15/15 [00:00<00:00, 15001.09it/s]
ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.
Traceback (most recent call last):
  File "D:\Downloads\lama-cleaner-win\lama-cleaner\installer\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "D:\Downloads\lama-cleaner-win\lama-cleaner\installer\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\Downloads\lama-cleaner-win\lama-cleaner\installer\Scripts\lama-cleaner.exe\__main__.py", line 7, in <module>
  File "D:\Downloads\lama-cleaner-win\lama-cleaner\installer\lib\site-packages\lama_cleaner\__init__.py", line 9, in entry_point
    main(args)
  File "D:\Downloads\lama-cleaner-win\lama-cleaner\installer\lib\site-packages\lama_cleaner\server.py", line 233, in main
    ui = FlaskUI(
TypeError: FlaskUI.__init__() got an unexpected keyword argument 'host'

**Comment by Sanster** [comment]
```bash
TypeError: FlaskUI.init() got an unexpected keyword argument 'host'
```

[flaskwebgui](https://github.com/ClimenteA/flaskwebgui) make some breaking change in version 1.0.0. Fixed in `lama-cleaner==0.25.0`, rerun `win_config.bat` will install the newest version.

**Comment by ffdown** [comment]
> ```shell
> TypeError: FlaskUI.init() got an unexpected keyword argument 'host'
> ```
> 
> [flaskwebgui](https://github.com/ClimenteA/flaskwebgui) make some breaking change in version 1.0.0. Fixed in `lama-cleaner==0.25.0`, rerun `win_config.bat` will install the newest version.


```
0.25.0

• remove topch.amp. autocast in sd1.5, гип 25% faster(tested оп 3080)
• remove Stpength from the sd side рапе], it has nothing to do with inpainting
• fix the flaskwebgui version

Assets

Source code (zip)
Source code (tar.gz)
```
where exe?

**Comment by DaBenjamins** [comment]
Same thing happens to me I get this when running it in CMD

------------------Environment information-------------------
git version 2.34.1.windows.1
conda 22.11.1
/c/Tools/lama-cleaner/installer/python
Python 3.10.6
/c/Tools/lama-cleaner/installer/Scripts/pip
pip 22.2.2 from C:\Tools\lama-cleaner\installer\lib\site-packages\pip (python 3.10)
diffusers              0.10.2
lama-cleaner           0.30.0
opencv-python          4.6.0.66
torch                  1.12.1+cu113
------------------------------------------------------------
[20:47:29] INFO     Load config from config.json                                                            tasks.py:111
           INFO     Using random port: 59820                                                                tasks.py:120
